CMakeLists.txt

```diff
@@ -8,7 +8,9 @@ set_property (GLOBAL PROPERTY USE_FOLDERS ON)
 
 set (CMAKE_CONFIGURATION_TYPES Debug;Release;RelWithDebInfo)
 set (AC_API_DEVKIT_DIR ${API_DEVKIT_DIR} CACHE PATH "API DevKit directory.")
-set (AC_ADDON_NAME "ExampleAddOn" CACHE STRING "Add-On name.")
+if (NOT DEFINED AC_ADDON_NAME)
```

Comment:
I guess it can be a cache variable like the others:
```
set (AC_ADDON_NAME "ExampleAddOn" CACHE STRING "Add-On name.")
```

Reply:
We do not need to set it in the CMakeLists file, we decided that AC_ADDON_NAME is a mandatory argument for generating the solution. This set and the if should get removed according to the latest Tools repository changes, and the readme should be modified accordingly.

Reply:
Fixed